src/dvr/dvr_db.c

```diff
@@ -3230,6 +3230,27 @@ dvr_entry_class_disp_extratext_get(void *o)
   return &prop_ptr;
 }
 
+static int
+dvr_entry_class_disp_extratext_set(void *o, const void *v)
+{
+  dvr_entry_t *de = (dvr_entry_t *)o;
+  const char *lang = idnode_lang(o);
+  v = tvh_str_default(v, "?????");
+
+  if (de->de_subtitle && lang_str_get(de->de_subtitle, lang)[0] != '\0'){
```

Comment:
I don't know if this is the best method to do the compare "lang_str_get(de->de_subtitle, lang)[0]" but it works.